.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
sets up vs code so that unit testing "just works" and tests can be stepped through.